FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Taras Zinchenko on 16.08.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImagesMapper {
```

Comment:
This class is not intended to be subclassed so it can be marked as `final`.

```suggestion
final class FeedImagesMapper {
```